roles/openshift_control_plane/files/scripts/docker/master-logs

```diff
@@ -17,13 +17,12 @@ for type in "${types[@]}"; do
   fi
 done
 
-# TODO: move to cri-ctl
-# TODO: short term hack for cri-o
+pod=$(crictl pods -l -q --label "openshift.io/component=${1}" --label "io.kubernetes.container.name=POD" 2>/dev/null)
+uid=$(crictl inspectp ${pod} 2>/dev/null | python -c 'import sys, json; print json.load(sys.stdin)["status"]["labels"]["io.kubernetes.pod.uid"]')
```

Comment:
we can drop podman in place of docker, IIRC.  No need for this inline python.

Reply:
Podman isn't integrated with cri-o, crictl is the only way for now